packages/office-ui-fabric-react/src/components/OverflowSet/OverflowSet.tsx

```diff
@@ -103,11 +149,88 @@ export class OverflowSet extends BaseComponent<IOverflowSetProps, {}> implements
 
   private _onRenderOverflowButtonWrapper = (items: any[]): JSX.Element => {
     const wrapperDivProps: React.HTMLProps<HTMLDivElement> = { className: css('ms-OverflowSet-overflowButton', styles.item) };
+    const overflowKeytipSequences = this.props.keytipSequences;
+    const itemSubMenuProvider = this.props.itemSubMenuProvider;
+
+    // Register all persisted keytips and save
+    if (overflowKeytipSequences) {
+      items.forEach((overflowItem) => {
+        const keytip = (overflowItem as IOverflowSetItemProps).keytipProps;
+        if (keytip) {
+          // Create persisted keytip
+          // TODO: test a persisted keytip that has a corresponding disabled button
```

Comment:
There are few scattered TODOs that are small and will be resolved before final checkin